### PR TITLE
Update disable-common.inc, disable-programs.inc.

### PR DESCRIPTION
**Pull request by rusty-snake**
Add some missing paths to disable-programs.inc and add some paths to disable-common.inc.
<br><br>
By the way, what about the paths of programs for which there is no profile, can you simply add them? (e.g. zim, tilix, aqemu, eclipse,...)

### CODE REVIEW

File: etc/disable-common.inc [hunk 1]
```diff
@@ -268,6 +268,9 @@ read-only ${HOME}/.gem
 read-only ${HOME}/.luarocks
 read-only ${HOME}/.npm-packages
 read-only ${HOME}/bin
+read-only ${HOME}/.bin
+read-only ${HOME}/.local/bin
+read-only ${HOME}/.cargo/bin
```
Comment (SkewedZeppelin) [inline etc/disable-common.inc]: why not just make all of cargo readonly?

File: etc/disable-programs.inc [hunk 2]
```diff
@@ -452,6 +453,7 @@ blacklist ${HOME}/.local/share/ocenaudio
 blacklist ${HOME}/.local/share/okular
 blacklist ${HOME}/.local/share/orage
 blacklist ${HOME}/.local/share/org.kde.gwenview
+blacklist ${HOME}/.local/share/rhythmbox
```
Comment (SkewedZeppelin) [inline etc/disable-programs.inc]: rhythmbox.profile will need to be updated

**Comment by SkewedZeppelin** [comment]
> By the way, what about the paths of programs for which there is no profile, can you simply add them? 

With care, yes.

File: etc/disable-programs.inc (hunk 2 above)
Comment (rusty-snake) [reply etc/disable-programs.inc]: Upps Forgotten, coming soon.

File: etc/disable-common.inc (hunk 1 above)
Comment (rusty-snake) [reply etc/disable-common.inc]: Idea:
```
$ ls .cargo
bin/  config  env  registry/
```
config and registry are only needed by cargo it self, so we can blacklist it.
env is maybe used by some shell config files (e.g. .profile, .bashrc,...). only read-only.

File: etc/disable-programs.inc [hunk 3]
```diff
@@ -395,6 +395,7 @@ blacklist ${HOME}/.local/share/akregator
 blacklist ${HOME}/.local/share/apps/korganizer
 blacklist ${HOME}/.local/share/aspyr-media
 blacklist ${HOME}/.local/share/baloo
+blacklist ${HOME}/.local/share/bibletime
```
Comment (rusty-snake) [inline etc/disable-programs.inc]: Fix in bibletime.profile comming soon.

File: etc/disable-common.inc (hunk 1 above)
Comment (SkewedZeppelin) [reply etc/disable-common.inc]: I don't rust, so I'll trust your judgement.

File: etc/disable-programs.inc (hunk 2 above)
Comment (rusty-snake) [reply etc/disable-programs.inc]: Sorry, but now is there

File: etc/disable-programs.inc (hunk 3 above)
Comment (rusty-snake) [reply etc/disable-programs.inc]: Fixed.

File: etc/disable-common.inc (hunk 1 above)
Comment (rusty-snake) [reply etc/disable-common.inc]: OK, after a short look, if we blacklist `registry` we muss update atom, gnome-builder, bracktes, maybe vim and emacs, in the future geany. If this is ok i will update these profiles tomorrow. It's getting late for me ;)

Comment (rusty-snake) [reply etc/disable-common.inc]: OK, I've made some changes.

**Comment by SkewedZeppelin** [comment]
Merged, thanks!

**Pull request merged**